test/falcor/deref/deref.errors.spec.js

```diff
@@ -35,7 +35,7 @@ describe('Error cases', function() {
                     get([0, 0, 'item', 'title'])).
                     doAction(onNext, function(err) {
                         expect(onNext.callCount).to.equal(1);
-                        expect(err.message).to.equals(InvalidModelError.message);
+                        expect(err.message).to.equals("The boundPath of the model is not valid since a value or error was found before the path end.");
```

Comment:
I think we should only check for `err.name` in tests, because that maps 1:1 with the error having the right `err.message`. simply delete the testing for `err.message` everywhere

Reply:
So change

```js
expect(err.message).to.equals("The boundPath of the model is not valid since a value or error was found before the path end.");
```

to

```js
expect(err.message)
```

?


Or delete it completely?

Reply:
delete completely